src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -696,4 +698,97 @@ public void setApplicationProtocols(String[] protocols) {
         }
         applicationProtocols = tempProtocols;
     }
+
+    /**
+     * Returns a prioritized array of signature scheme names that can be used
+     * over the SSL/TLS/DTLS protocols.
+     * <p>
+     * Note that the standard list of signature scheme names may be found in
+     * the <a href=
+     * "{@docRoot}/../specs/security/standard-names.html#signature-schemes">
+     * Signature Schemes</a> section of the Java Security Standard Algorithm
+     * Names Specification. Providers may support signature schemes not found
+     * in this list or might not use the recommended name for a certain
+     * signature scheme.
+     * <p>
+     * The returned array could be {@code null}, in which case the underlying
+     * provider-specific default signature schemes will be used over the
+     * SSL/TLS/DTLS connections.
+     * <p>
+     * The returned array could be empty (zero-length), in which case the
+     * signature scheme negotiation mechanism is turned off for SSL/TLS/DTLS
+     * protocols, and the connections may not be able estabilished if the
+     * negotiation mechanism is required by a certain SSL/TLS/DTLS protocol.
+     * <p>
+     * The returned array could be neither {@code null} nor empty (zero-length),
```

Comment:
Suggest rewording as "If the returned array is not {@code null} or empty (zero-length), then the signature schemes .."